server/utils/nunjucksSetup.ts

```diff
@@ -92,6 +92,7 @@ export default function nunjucksSetup(app: express.Express, path: pathModule.Pla
   njkEnv.addGlobal('dateFieldValues', dateFieldValues)
   njkEnv.addGlobal('formatDate', DateFormats.isoDateToUIDate)
   njkEnv.addGlobal('formatDateTime', DateFormats.isoDateTimeToUIDateTime)
+  njkEnv.addFilter('mojDate', DateFormats.isoDateTimeToUIDateTime)
```

Comment:
So is this used internally by the timeline component?

Reply:
yes https://github.com/ministryofjustice/moj-frontend/blob/main/src/moj/components/timeline/template.njk#L18